internal/controller/traps/filesystoken/utils_test.go

```diff
@@ -27,9 +27,9 @@ import (
 var (
 	containerSelectorValues = []string{
 		"name",
-		"namewithwildcard.*",
-		"namewithwildcard?",
-		".*",
+		"glob:namewithwildcard.*",
```

Comment:
[ISSUE] I think the dot is a left-over that is not needed for the glob syntax.

```suggestion
		"glob:namewithwildcard*",
```